aggregators/magpie/index.ts

```diff
@@ -92,6 +92,16 @@ const fetch = async (_t: number, _: ChainBlocks, {chain, startOfDay}: FetchOptio
           runAtCurrTime: true,
           start: '2022-09-08',
         },
+        [CHAIN.SONIC]: {
+          fetch: fetch,
+          runAtCurrTime: true,
+          start: 1735217146,
+        },
+        [CHAIN.ZKSYNC]: {
```

Comment:
is it zksync or zk sync era ?, if correct could you check error ?

Reply:
@dtmkeng I updated it